mesos/mesos.go

```diff
@@ -121,6 +137,7 @@ func (m *Mesos) GetMetricTypes(cfg plugin.ConfigType) ([]plugin.MetricType, erro
 
 		for _, key := range agent_stats {
 			namespace := core.NewNamespace(pluginVendor, pluginName, "agent").
+				AddStaticElement("task").
```

Comment:
And what is executor ? task .. in general it is task in mesos which is launched in framework via executor. Because, if we follow naming, then I'd need to add /framework/<fid>/executor/<eid>. Together that is task - so /task/<fid>/<eid>